docs/v1/api.md

```diff
@@ -1,63 +1,96 @@
 # V1 API
 
-This document contains resource types and their standard usage in v1 service
-catalog. Although this API will be implemented in Kubernetes, other systems
-are not precluded from implementing it as well.
+This document describes the API resource types and usage in the v1 service
+catalog. Although this API will be implemented in a Kubernetes repo, other
+systems are not precluded from implementing it as well.
 
 # Resource Types
 
 This section lists descriptions of the resources in the service catalog API.
 
 *__Note:__ All names herein are tentative, and should be considered placeholders
-and used as placeholders for purposes of discussion only. This note will be
-removed when names are finalized.*
+for purposes of discussion. This note will be removed when names are finalized.*
 
-## `Broker`
+## `Broker` Resource
 
-This resource is created by an administrator to instruct the service catalog's
-controller event loop to do the following:
+A `Broker` represents an entity that provides service classes for use in the
+catalog.
 
-1. Make a request against a given CF service broker's catalog endpoint
+An administrator creates an instance of the `Broker` resource to indicate their
+intent to make the service classes provided by that broker available in the
+catalog.
+
+The catalog controller has a watch on the `Broker` resource.  When the
+controller receives an `ADD` watch event for a new `Broker`, it contacts the
+broker to determine what service classes the broker offers:
+
+1. Makes a request against the given service broker's catalog endpoint
    (`GET /v2/catalog`)
-2. Translate the response to a list of `ServiceClass`es
-3. Write each translated `ServiceClass` to stable storage
+2. Translates the response to a list of `ServiceClass`
+3. Creates new `ServiceClass` instances in the API server
 
-*TODO: should we call out what happens when a `Broker` resource is deleted?*
+*TODO: What happens when a `Broker` resource is deleted?*
 
-## `ServiceClass`
+## `ServiceClass` Resource
+
+A `ServiceClass` represents an offering in the service catalog.  A
+`ServiceClass` is not usable directly; an `Instance` of a `ServiceClass` must be
+created to be consumed by an application.
 
 This resource is created by the service catalog's controller event loop after
```

Comment:
ok for now, I'm not a fan of repeating stuff since its asking for things to get out of sync. So we shouldn't repeat what was stated in the previous section.